src/sentry/static/sentry/app/api.tsx

```diff
@@ -440,7 +445,7 @@ export class Client {
 
         // Try to get text out of the response no matter the status
         try {
-          responseText = await response.text();
+          responseText = await responseClone.text();
```

Comment:
Turns out this was always undefined because the body was already drained. i.e. `response.bodyUsed = true` by the time this line was reached.

Reply:
To elaborate, this was throwing an error because the body was already read. The error was swallowed.